src/index.html

```diff
@@ -16,11 +16,31 @@
 </head>
 
 <body>
+  <div class="header">
+  <h1><a class="name" href="https://github.com/Stephenich/stephenich.github.io">Stehen Gurney</a></h1>
+    <img src="images/githubIcon.svg" alt="GitHub Logo">
+  </div>
+   <header> <div class="menu">
+      <ul>
+        <li>Home</li>
+        <li>GitHub</li>
+      </ul>
+    </div>
+  </header>
 
-  <p>Hello world! This is my HTML will this work??.</p>
-  <img src="images/bigben.jpg" alt="bigben" width="400" height="400"> 
-  <img src="images/london.jpg" alt="london" width="400" height="400">
+  <div>
```

Comment:
you could use `<article>`  or even `<main>`

Reply:
I'll look into the difference of these tags, and change it accordingly